### ISSUE TITLE
Sails keeps running on the background even if I have stoppped it

### ISSUE THREAD

**Issue by sahanDissanayake** [balderdashy/sails#2288]
Hi,

I just installed my app on AWS EC2. And this is crazy. The app just keeps running after I have taken down the server.

Port : 80
Sails : 0.10.5
If I need to stop the app from running I have to reboot the server. 

( Is anyone using EC2 Ubuntu server on port 80 to run a SailsJS app ? )

Please let me know about this

**Comment by Dallas62** [comment]
Is that solved?

**Comment by CWyrtzen** [comment]
Has this been resolved?

**Comment by sailsbot** [comment]
Thanks for posting, @sahanDissanayake.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
